Makefile

```diff
@@ -135,6 +143,15 @@ test-integration: manifests generate fmt vet envtest ginkgo ## Run tests.
 	KUBEBUILDER_ASSETS="$(shell $(ENVTEST) --arch=amd64 use $(ENVTEST_K8S_VERSION) -p path)" \
 	$(GINKGO) --junit-report=junit.xml --output-dir=$(ARTIFACTS) -v $(INTEGRATION_TARGET)
 
+USE_EXISTING_CLUSTER ?= false
+.PHONY: test-e2e
+test-e2e: USE_EXISTING_CLUSTER=true
+test-e2e: test-e2e-kind
```

Comment:
Given that we will follow up on running tests that don't imply kind, let's remove these lines.